### PR TITLE
Sort & update publicise languages with italian and portuguese additions

### LINKED ISSUE #2038: Updated publicise tabs to latest languages on snap-store badges repo
Make sure the publicise tab includes the following languages:

- English
- Deutsch
- Japanese
- French
- Spanish
- Chinese(Taiwan)
- Russian
- Portuguese
- Italian (Italy)

[Badges repo assets can be found here](https://github.com/snapcore/snap-store-badges)

### PR DESCRIPTION
**Pull request by therealjuan**
Fixes #2038 

### QA Steps

1. Pull the branch or run the demo service
2. Log in and check any snap you own on its `/publicise` page
3. Check on the Snap Store badges that Italian and Portuguese are available
4. #winning

### CODE REVIEW

**Comment by kwm14** [comment]
**No labels. No review** 
_Luke, May 2019_

**Comment by Lukewh** [comment]
**No labels. No review.** 
_Luke, May 2019_
_Karlos, June 2019_

File: webapp/publisher/snaps/views.py [hunk 1]
```diff
@@ -1117,14 +1117,16 @@ def get_publicise(snap_name):
...
             "title": "Français",
             "text": "Installer à partir du Snap Store",
         },
-        "jp": {"title": "日本語", "text": "Snap Store から入手ください"},
-        "ru": {"title": "русский язык", "text": "Загрузите из Snap Store"},
+        "es": {"title": "Español", "text": "Instalar desde Snap Store"},
         "tw": {"title": "中文（台灣）", "text": "安裝軟體敬請移駕 Snap Store"},
+        "ru": {"title": "русский язык", "text": "Загрузите из Snap Store"},
+        "pt": {"title": "Português", "text": "Disponível na Snap Store"},
+        "it": {"title": "Italiano", "text": "Scarica dallo Snap Store"},
```
Comment (bartaz) [inline webapp/publisher/snaps/views.py]: Are these in any particular order?
I think they used to be alphabetical (with English as default on top), now order shifted and seems random.

**Comment by bartaz** [comment]
Apart from listing new languages we are also linking to .zip with all the badges. Currently linked version is 1.2 which seems out of date. There is no other version.

I think it should be updated as well at the same time.

**Comment by therealjuan** [comment]
- Created a new release v1.3 in badges to link from the publicise tabs
- Sorted alphabetically all languages
- Made english default

**Comment by therealjuan** [comment]
**Labels. Labels. Labels everywhere!** 
_Juan_, Juan 2019_

**Comment by bartaz** [comment]
Can we add Polish badges as well while we are at it? They are in the repo and .zip

**Pull request merged**